src/index.html

```diff
@@ -1,12 +1,373 @@
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body body">
+    <header class="body__header header">
+      <a href="#home">
+        <img
+             src="./images/logo.png"
+             alt="Moyo"
+             class="header__logo"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#nav-link"
+               class="nav__link is-active">Apple
+            </a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#"
+               class="nav__link"
+               data-qa="nav-hover">Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+      <!-- <div class="top-bar__icons">
+        <a href="#menu" class="icon icon--menu"></a>
+      </div>
+      <aside class="header__menu menu" id="menu">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img
+              src="./images/logo.png"
+              alt="logo"
+              class="top-bar__logo"
+            >
+          </a>
+          <a
+            href="#"
+            class="icon icon--close"
+          >
+          </a>
+        </div>
+        <div class="menu__content">
+
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#" class="nav__link">Home</a>
+              </li>
+              <li class="nav__item">
+                <a href="#about-us" class="nav__link">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#compare-bikes" class="nav__link">Compare Bikes</a>
+              </li>
+              <li class="nav__item">
+                <a href="#details" class="nav__link">Details</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contacts" class="nav__link">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </aside> -->
+    </header>
+
+    <main class="body__main main">
+      <div class="card-container">
+        <div class="card-container__cards cards" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            class="cards__laptop"
+            alt="APPLE A1419 iMac 27"
+          >
+          </img>
```

Comment:
```suggestion
          <img
            src="./images/imac.jpeg"
            class="cards__laptop"
            alt="APPLE A1419 iMac 27"
          >
```
Img tag is self-closing